docs/how_to_guides/documentation.md

```
@@ -151,5 +151,11 @@ We may decide to check the documentation with a "linter" which is designed to ke
 One option is [markdownlint-cli](https://github.com/igorshubovych/markdownlint-cli), which uses David Anson's [NodeJS-based markdownlint](https://github.com/DavidAnson/markdownlint), which these days seems to be more popular than the [ruby-based markdownlint](https://github.com/markdownlint/markdownlint).
 A `.markdownlint.jsonc` file has been added that configures the same markdownlint program (basically to ignore the rule about long lines, since we are using ventilated prose).
 
+## Doc Philosphy
+
+We try to use [Diátaxis](https://diataxis.fr/).
+This is also highly related: https://docsfordevelopers.com/
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)